.github/workflows/build-run-kernel-snake.yml

```diff
@@ -16,8 +16,8 @@ jobs:
           cache: true
           cache-write: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
       - name: Build and run kernels
-        run: pixi run snakemake -k -j `nproc`
+        run: pixi run snakemake -k -j `nproc` all
```

Comment:
could we maybe make a pixi task for this such that we can do `pixi make`? would be convenient
not needed here though, just a thought

Reply:
what is a pixi-task? :D